scripts/l2SPOdeploy.ts

```diff
@@ -0,0 +1,54 @@
+import l2config from './l2config';
+import { getAllContractAddresses } from '@scripts/utils/loadContracts';
+import { ethers } from 'hardhat';
+import '@nomiclabs/hardhat-ethers';
+import { expect } from 'chai';
+
+const {
+  /// bls cpi-u inflation data
+  L2_ARBITRUM_PREVIOUS_MONTH,
+  L2_ARBITRUM_CURRENT_MONTH,
+
+  /// L2 chainlink
+  STARTING_L2_ORACLE_PRICE,
+  ACTUAL_START_TIME,
+  L2_ARBITRUM_JOB_ID,
+  L2_ARBITRUM_CHAINLINK_FEE,
+  L2_ARBITRUM_CHAINLINK_TOKEN
+} = l2config;
+
+async function deploy() {
+  /// -------- System Deployment --------
+
+  const addresses = await getAllContractAddresses();
+  const L2ScalingPriceOracleFactory = await ethers.getContractFactory('L2ScalingPriceOracle');
+
+  const scalingPriceOracle = await L2ScalingPriceOracleFactory.deploy(
+    addresses.arbitrumFiewsChainlinkOracle,
```

Comment:
arbitrumFiews?

Reply:
Yeah, this is the chainlink data provider that will serve data to the Scaling Price Oracle on Arbitrum.